gcp_deepvariant_runner.py

```diff
@@ -1040,10 +1102,57 @@ def run(argv=None):
             'jobs. By default, the pipeline runs all 3 jobs (make_examples, '
             'call_variants, postprocess_variants) in sequence. '
             'This option may be used to run parts of the pipeline.'))
+  parser.add_argument(
+      '--stop_collecting_anonymous_usage_metrics',
```

Comment:
I think add the argument `collecting_anonymous_usage_metrics` would be easier to read. Especially it is used as `if not pipeline_args.stop_collecting_anonymous_usage_metrics:`. And the `action` and `help` seem not relevant?